pages/posts/index.tsx

```diff
@@ -0,0 +1,107 @@
+/* Copyright 2020 the Deno authors. All rights reserved. MIT license. */
+
+import React from "react";
+import Head from "next/head";
+import { GetStaticProps } from "next";
+
+import { promises } from "fs";
+
+import Header from "../../components/Header";
+import Footer from "../../components/Footer";
+import { join } from "path";
+import Link from "next/link";
+
+interface PostMeta {
+  id: string;
+  title: string;
+  author: string;
+  publish_date: string;
+  images: Array<{
+    image: string;
+    width: number;
+    height: number;
+    className: string;
+    preview: boolean;
+  }>;
+  snippet: string;
+}
+
+interface Props {
+  posts: PostMeta[];
+}
+
+const PostsIndexPage = (props: Props) => {
+  return (
+    <>
+      <Head>
+        <title>News | Deno</title>
+      </Head>
+      <Header />
+      <div className="bg-white pt-8 pb-20 px-4 sm:px-6 lg:pt-8 lg:pb-28 lg:px-8">
+        <div className="relative max-w-screen-lg mx-auto">
+          <div className="border-b-2 border-gray-100 pb-10">
+            <h2 className="text-4xl font-bold tracking-tight">News</h2>
+            <div className="mt-3 sm:mt-4">
+              <p className="text-xl leading-7 text-gray-500">
+                Status updates about the Deno project from the Deno team.
+              </p>
+            </div>
+          </div>
+          <div className="mt-6 grid gap-16 lg:grid-cols-2 lg:col-gap-5 lg:row-gap-12">
+            {props.posts.map((post) => {
+              const date = new Date(post.publish_date);
+              const format = new Intl.DateTimeFormat(undefined, {
+                month: "long",
+                day: "numeric",
+                year: "numeric",
+              });
+              return (
+                <div key={post.id}>
+                  <p className="text-sm leading-5 text-gray-500">
+                    <time dateTime="2020-03-16">{format.format(date)}</time>
```

Comment:
Why is the `dateTime` attribute's value fixed? And also it should be `datetime`.